packages/p2p/src/components/modal-manager/modals/order-details-confirm-modal/order-details-confirm-modal.tsx

```diff
@@ -1,38 +1,36 @@
 import React from 'react';
 import { useStores } from 'Stores';
 import { Button, Modal, Text } from '@deriv/components';
-import { isMobile, formatMoney } from '@deriv/shared';
+import { formatMoney, TFile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import FileUploaderComponent from 'Components/file-uploader-component';
+//Todo: change the below line when changing the base branch
```

Comment:
`nit`: i think we can be more specific here with this todo, and just say when merging with our refactor reusable components branch